src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -73,6 +75,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $marking->mark($place);
         }
 
-        $output->writeln($dumper->dump($workflow->getDefinition(), $marking));
+        $options = array();
+        $label = $input->getOption('label');
+        if (null !== $label && '' !== trim($label)) {
+            $options = array('graph' => array('label' => $input->getOption('label')));
```

Comment:
You should be using $label instead of $input->getOption('label').

Reply:
thx, it's done